From 51d7e518fd026469ba31f503d8e599894b76cff5 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Wed, 7 Sep 2016 23:27:07 -0400
Subject: [PATCH] disable dexopt profile usage

Profiling is disabled, but make sure that profiles are not used at all.

Signed-off-by: anupritaisno1 <www.anuprita804@gmail.com>
---
 target/product/runtime_libart.mk | 11 ++++-------
 1 file changed, 4 insertions(+), 7 deletions(-)

diff --git a/target/product/runtime_libart.mk b/target/product/runtime_libart.mk
index 031e7b827..ec0f89b81 100644
--- a/target/product/runtime_libart.mk
+++ b/target/product/runtime_libart.mk
@@ -107,22 +107,19 @@ else
         pm.dexopt.boot-after-ota?=verify
 endif
 
-# The install filter is speed-profile in order to enable the use of
-# profiles from the dex metadata files. Note that if a profile is not provided
-# or if it is empty speed-profile is equivalent to (quicken + empty app image).
 # Note that `cmdline` is not strictly needed but it simplifies the management
 # of compilation reason in the platform (as we have a unified, single path,
 # without exceptions).
 PRODUCT_SYSTEM_PROPERTIES += \
     pm.dexopt.post-boot?=extract \
-    pm.dexopt.install?=speed-profile \
+    pm.dexopt.install?=speed \
     pm.dexopt.install-fast?=skip \
-    pm.dexopt.install-bulk?=speed-profile \
+    pm.dexopt.install-bulk?=speed \
     pm.dexopt.install-bulk-secondary?=verify \
     pm.dexopt.install-bulk-downgraded?=verify \
     pm.dexopt.install-bulk-secondary-downgraded?=extract \
-    pm.dexopt.bg-dexopt?=speed-profile \
-    pm.dexopt.ab-ota?=speed-profile \
+    pm.dexopt.bg-dexopt?=speed \
+    pm.dexopt.ab-ota?=speed \
     pm.dexopt.inactive?=verify \
     pm.dexopt.cmdline?=verify \
     pm.dexopt.shared?=speed
-- 
2.17.1

